src/constraints/PositiveSemidefiniteConeConstraint.jl

```diff
@@ -57,10 +57,6 @@ function populate_dual!(
     return
 end
 
-# TODO: Remove isposdef, change tests to use in. Update documentation and
-# notebooks
-LinearAlgebra.isposdef(x::AbstractExpr) = in(x, :SDP)
-
```

Comment:
Actually, I prefer `isposdef` than `x in :semidefinite`, maybe we can deprecate the `in :SDP` instead ?

Reply:
I don't really have an opinion. I'm just following the comment :smile:

`isposdef` seems more 'convex'.

Reply:
Ah yes indeed, I didn't see that comment. Indeed, the `Symbol` approach isn't particularly convenient since the typos in the symbol aren't checked at compile time (this is why we switched to `Enum` in JuMP v0.19). Also, this is actually the only symbol supported, it's not like other symbols are supported so it's kind of weird.

Reply:
This seems to date back to @karanveerm's PR in 2015 https://github.com/jump-dev/Convex.jl/pull/50. I think we can keep `isposdef` for now

Reply:
I'm happy to swap to deprecating the `in` syntax. we don't need three ways to specify a PSD matrix.